src/plugin/timezone/index.js

```diff
@@ -96,7 +96,8 @@ export default (o, c, d) => {
     const oldOffset = this.utcOffset()
     const date = this.toDate()
     const target = date.toLocaleString('en-US', { timeZone: timezone })
-    const diff = Math.round((date - new Date(target)) / 1000 / 60)
+    const targetDate = d(target, 'M/D/YYYY H:mm:ss A')
```

Comment:
can we use d(target) directlly?

Reply:
Actually we can't use it, because `parseDate` method returns `new Date` (https://github.com/iamkun/dayjs/blob/dev/src/index.js#L79), which is the reason of this PR and in case of using `customParseFormat` plugin without passing `format` argument it calls original `parseDate` method under the hood https://github.com/iamkun/dayjs/blob/dev/src/plugin/customParseFormat/index.js#L257. So my fix will work only when using `customParseFormat` plugin, what do you think about it?

Reply:
True, that's my concern. If we implement it in this way, the timezone plugin must rely on customParseFormat plugin to work that our users may get confused

Reply:
I got your point. Maybe I can find a better solution to calculate timezone offset not to depend on some APIs support.

Reply:
@iamkun Could you please check the new solution? I am using `formatToParts` method of `Intl.DateTimeFormat` instance and getting year, month, day, etc. from its result. What do you think? I tried this in my React Native environment and it works perfectly. All tests are passed correctly as well.